XamlControlsGallery/ControlPages/InfoBarPage.xaml

```diff
@@ -0,0 +1,122 @@
+                <local:ControlExampleSubstitution x:Name="DisplayButton" Key="DisplayButton" Value=""/>
+            </local:ControlExample.Substitutions>
+            <local:ControlExample.Options>
+                <StackPanel>
+                    <CheckBox x:Name="IsOpenCheckBox2" AutomationProperties.Name="IsOpenCheckBox2" Content="Is Open" IsChecked="{x:Bind TestInfoBar2.IsOpen, Mode=TwoWay}"/>
+                    <ComboBox x:Name="MessageComboBox" AutomationProperties.Name="MessageComboBox" Header="Message Length" SelectionChanged="MessageComboBox_SelectionChanged" SelectedIndex="1">
+                        <ComboBoxItem Content="Short" />
+                        <ComboBoxItem Content="Long" />
+                    </ComboBox>
+                    <ComboBox x:Name="ActionButtonComboBox" AutomationProperties.Name="ActionButtonComboBox" Header="Action Button" SelectionChanged="ActionButtonComboBox_SelectionChanged" SelectedIndex="0">
+                        <ComboBoxItem Content="None" />
+                        <ComboBoxItem Content="Button" />
+                        <ComboBoxItem Content="Hyperlink" />
+                    </ComboBox>
+                </StackPanel>
+            </local:ControlExample.Options>
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;InfoBar 
+    IsOpen="$(IsOpen)"
+    Title="Title"
+    Message="$(DisplayMessage)"
+    $(DisplayButton)/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+        </local:ControlExample>
+        <local:ControlExample x:Name="Example3" HeaderText="A closable InfoBar with options to display the close button and icon">
+            <local:ControlExample.Example>
+                <controls:InfoBar x:Name="TestInfoBar3" AutomationProperties.Name="TestInfoBar3"
+                    IsOpen="True"
+                    IsIconVisible="True"
+                    IsClosable="True"
+                    Title="Title"
+                    Message="Essential app message for your users to be informed of, acknowledge, or take action on." />
+            </local:ControlExample.Example>
+            <local:ControlExample.Options>
+                <StackPanel>
+                    <CheckBox x:Name="IsOpenCheckBox3" AutomationProperties.Name="IsOpenCheckBox3" Content="Is Open" IsChecked="{x:Bind TestInfoBar3.IsOpen, Mode=TwoWay}"/>
+                    <CheckBox x:Name="IsIconVisibleCheckBox" AutomationProperties.Name="IsIconVisibleCheckBox" Content="Is Icon Visible" IsChecked="{x:Bind TestInfoBar3.IsIconVisible, Mode=TwoWay}"/>
+                    <CheckBox x:Name="IsClosableCheckBox" AutomationProperties.Name="IsClosableCheckBox" Content="Is Closable" IsChecked="{x:Bind TestInfoBar3.IsClosable, Mode=TwoWay}"/>
+                </StackPanel>
+            </local:ControlExample.Options>
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;InfoBar 
```

Comment:
These examples are probably long enough to warrant their own .txt files rather than being inlined here.

Reply:
After discussion, these are unneeded.